pkg/click/click

```diff
@@ -175,7 +190,7 @@ shift $((OPTIND-1))
 
 # check for mandatory input
 if [[ -n $INPUT ]]; then
-    MIN_INPUT=1
+    MIN_INPUT=4
```

Comment:
What's the reason for this? Under no circumstances is the minimum number of inputs to `click` ever 4.

Reply:
I'm not sure why i changed it to 4 I must have done it absent minded. changed it back to 1